src/zcash/ZcashEngine.ts

```diff
@@ -42,7 +42,11 @@ export class ZcashEngine extends CurrencyEngine<
   synchronizerStatus!: ZcashSynchronizerStatus
   availableZatoshi!: string
   initializer!: ZcashInitializerConfig
-  progressRatio!: number
+  progressRatio!: {
+    percent: number
+    lastUpdate: number
```

Comment:
Should we store this as an ISO date string?

Reply:
I think a number is appropriate since we're just seeing if a second has passed to prevent to many callbacks